### DISCUSSION TITLE
Mantine V7.2.3, Will this work?

### DISCUSSION THREAD

**Discussion by johnLamberts** [KevinVandy/mantine-react-table#234] [Q&A]
I just want to ask if the latest version of mantine will work on this library? Yes, the recommended version is v6 but I do want to integrate the latest version of mantine in this library. 

Hopefully, don't bash me. hehe, I am currently looking for other dev who is using this library with the latest version of mantine.

**Comment by coredevel** [comment]
I've just tried installing it and forced the install and for ref here are the output warnings which provide a good idea of what potential issues to keep an eye out for, if not providing a course for the upgrade path.

```
npm WARN using --force Recommended protections disabled.
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: mantine-react-table@1.3.4
npm WARN Found: @mantine/core@7.3.2
npm WARN node_modules/@mantine/core
npm WARN   peer @mantine/core@"7.3.2" from @mantine/carousel@7.3.2
npm WARN   node_modules/@mantine/carousel
npm WARN     @mantine/carousel@"^7.3.2" from the root project
npm WARN   9 more (@mantine/code-highlight, @mantine/dates, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer @mantine/core@"^6.0" from mantine-react-table@1.3.4
npm WARN node_modules/mantine-react-table
npm WARN   mantine-react-table@"*" from the root project
npm WARN 
npm WARN Conflicting peer dependency: @mantine/core@6.0.21
npm WARN node_modules/@mantine/core
npm WARN   peer @mantine/core@"^6.0" from mantine-react-table@1.3.4
npm WARN   node_modules/mantine-react-table
npm WARN     mantine-react-table@"*" from the root project
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: mantine-react-table@1.3.4
npm WARN Found: @mantine/dates@7.3.2
npm WARN node_modules/@mantine/dates
npm WARN   @mantine/dates@"^7.3.2" from the root project
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer @mantine/dates@"^6.0" from mantine-react-table@1.3.4
npm WARN node_modules/mantine-react-table
npm WARN   mantine-react-table@"*" from the root project
npm WARN 
npm WARN Conflicting peer dependency: @mantine/dates@6.0.21
npm WARN node_modules/@mantine/dates
npm WARN   peer @mantine/dates@"^6.0" from mantine-react-table@1.3.4
npm WARN   node_modules/mantine-react-table
npm WARN     mantine-react-table@"*" from the root project
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: mantine-react-table@1.3.4
npm WARN Found: @mantine/hooks@7.3.2
npm WARN node_modules/@mantine/hooks
npm WARN   peer @mantine/hooks@"7.3.2" from @mantine/carousel@7.3.2
npm WARN   node_modules/@mantine/carousel
npm WARN     @mantine/carousel@"^7.3.2" from the root project
npm WARN   10 more (@mantine/code-highlight, @mantine/core, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer @mantine/hooks@"^6.0" from mantine-react-table@1.3.4
npm WARN node_modules/mantine-react-table
npm WARN   mantine-react-table@"*" from the root project
npm WARN 
npm WARN Conflicting peer dependency: @mantine/hooks@6.0.21
npm WARN node_modules/@mantine/hooks
npm WARN   peer @mantine/hooks@"^6.0" from mantine-react-table@1.3.4
npm WARN   node_modules/mantine-react-table
npm WARN     mantine-react-table@"*" from the root project
```

**Comment by KevinVandy** [comment]
If you use the MRT v2 alpha, the minimum Mantine version in 7.3

**Reply by masterbater** [reply]
_Eventually. MRT v2 will be completely rewritten to use Mantine v7. This should be available by the end of 2023._
MRT v2 still in alpha, not yet beta. Does v2 could be release first quarter of 2024?

**Reply by coredevel** [reply]
I've been using MRT v2 and it works ok but there's some weirdness with the styling:

![image](https://github.com/KevinVandy/mantine-react-table/assets/138916/e2e5cc0d-6c9a-4f15-8278-a813bdb715e2)

This is a table inside the main component of the app shell.  Are there some styles which need importing?

**Reply by coredevel** [reply]
Also with this when there are 'excessive' cols it simply overflows and doesn't have an h-scroll

**Reply by KevinVandy** [reply]
Yeah, looks like you didn't import the MRT styles at all

**Reply by coredevel** [reply]
I thought so, where can they be found (or the relevant docs, sorry I couldn't see them either)

**Reply by coredevel** [reply]
Just had a look through some examples on the git repo to and couldn't see anywhere themes were being imported so I'm not quite sure what I'm missing 🤔

**Reply by KevinVandy** [reply]
Install instructions

**Reply by coredevel** [reply]
Still no luck unfortunately - and as per the install instructions FAQ style issue I've ensured mantine v7.3.1 is installed per MRT v2 package deps but that still hasn't imported the styles.

**Reply by coredevel** [reply]
(PS let me know if there are any contributions you need with the upgrade, always happy to help)

**Reply by coredevel** [reply]
Just realised I was reading the v1 docs :upside_down_face: The styles imported successfully :smile:

I've just joined the discord channel so sure I'll have some feedback to post over the next couple of weeks as I build out some ideas. Thanks again!
